packages/cli/test/commands/migrate/validate.test.ts

```diff
@@ -53,51 +53,35 @@ describe('Task: validate', () => {
   });
```

Comment:
project.dispose() is called here. Somehow the afterEach block is collapsed in git diff.